chimney-macro-commons/src/main/scala/io/scalaland/chimney/internal/compiletime/datatypes/ProductTypes.scala

```diff
@@ -167,27 +163,13 @@ object ProductTypes {
     def isMatching(value: String): Boolean = regexp.pattern.matcher(value).matches() // 2.12 doesn't have .matches
   }
 
-  def areNamesMatching(fromName: String, toName: String): Boolean =
-    fromName == toName || normalize(fromName) == normalize(toName)
-
   private val getAccessor = raw"(?i)get(.)(.*)".r
   private val isAccessor = raw"(?i)is(.)(.*)".r
```

Comment:
I think these 2 can be removed - if some piece of code needs to detect getters/setters explicitly (`ProductTypes`?) it can refer `TransformerNamesComparison.BeanAware` explictly. Then some other helper methods would have to be exposed there instead (`isGetterName`), this way we would not duplicate the code.

Or wait.

There is module dependency order. Nope. It would have to be defined in `ProductTypes` companion object, called explicitly by pieces of code that DO rely on `dropSet` explicitly, and `TransformerNamesComparison.BeanAware` would have to delegate to that implementation. A bit nasty. So these names would have to be modified slightly to stay informative, e.g. all `areNamesMatching`, `isGetterName` etc, get moved to some:

```scala
// companion
object ProductTypes {

  object BeanAware {
    
    // all these name methods
    def areNamesMatching = ...
  }
}
```

```scala
// in the macro code when hardcoded approach is needed
import ProductTypes.BeanAware
BeanAware.areNamesMatching(fromName, toName)
```

```scala
object TransformerNamesComparison {

  object BeanAware extends TransformerNamesComparison {

    private val impl = internal.compiletime.ProductNames.BeanAware

    def areNamesMatching(fromName, toName) = impl.areNamesMatching(fromName, toName)
  }
}
```

Reply:
Resolved on #478